src/d3a/models/strategy/__init__.py

```diff
@@ -469,6 +490,15 @@ def can_offer_be_posted(
         return self.offers.can_offer_be_posted(
             offer_energy, offer_price, available_energy, market, replace_existing=replace_existing)
 
+    def can_settlement_offer_be_posted(self, offer_energy, offer_price, market,
+                                       replace_existing=False):
+        if not self.state.should_post_offer(market.time_slot):
```

Comment:
As consensus is to add docstring for each newly added methods.

Reply:
Adding docstring